validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorTimingActions.java

```diff
@@ -97,8 +105,30 @@ public void onAttestationAggregationDue(final UInt64 slot) {
   }
 
   @Override
-  public void onAttesterSlashing(final AttesterSlashing attesterSlashing) {}
+  public void onAttesterSlashing(final AttesterSlashing attesterSlashing) {
+    delegates.forEach(delegates -> delegates.onAttesterSlashing(attesterSlashing));
+    final Set<UInt64> slashedIndices = attesterSlashing.getIntersectingValidatorIndices();
```

Comment:
please put this one inside `ifPresent` block too

Reply:
Done

Reply:
Added it tests too